EIPS/eip-1690.md

```diff
@@ -0,0 +1,174 @@
+---
+eip: 1690
+title: ERC-1690 - Contract Mortability Standard
```

Comment:
```suggestion
title: Contract Mortability Standard
```

Reply:
After changing this, the [`eip-validator` job](https://travis-ci.org/github/ethereum/EIPs/jobs/730176275) will fails.